### PR TITLE
Remove OnboardingDerivationPathSelect component from pages folder

### PR DESCRIPTION
**Pull request by 7alip**
1. I removed `OnboardingDerivationPathSelect` component which should not be in pages dir.
2. It dispatches `importLegacyKeyring` action twice.
https://github.com/tallycash/extension/blob/7f0b7cae6245ac95599a4eac7490de729936fbe8/ui/pages/Onboarding/OnboardingImportMetamask.tsx#L97-L98

### CODE REVIEW

**Review by henryboldi — CHANGES_REQUESTED** [review]
Thoughts on moving `ui/pages/Onboarding/OnboardingDerivationPathSelect.tsx` to `ui/components/Onboarding/OnboardingDerivationPathSelect.tsx`? That's where I'd put it. And I'd love to keep the separate component!

File: ui/pages/Onboarding/OnboardingImportMetamask.tsx [hunk 1]
```diff
@@ -94,7 +118,6 @@ export default function OnboardingImportMetamask(props: Props): ReactElement {
     const trimmedRecoveryPhrase = recoveryPhrase.trim()
     if (isValidMnemonic(trimmedRecoveryPhrase)) {
       setIsImporting(true)
-      dispatch(importLegacyKeyring({ mnemonic: recoveryPhrase.trim(), path }))
```
Comment (henryboldi) [inline ui/pages/Onboarding/OnboardingImportMetamask.tsx:97]: Oh jeez. Looks like resolving merge conflicts went wrong at some point. Thank you for splitting this!

**Review by henryboldi — APPROVED** [review]
Looks good! 😁

**Comment by Shadowfiend** [comment]
Force-merging here since the build error is build configuration-related (see #793).

Ran `yarn build` manually to confirm:

<details>
<pre>
yarn run v1.22.10
$ webpack --mode=production
 
              ui.js ⏤  902 kB (+48.7 kB)
      background.js ⏤  596 kB (+3.13 kB)
   background-ui.js ⏤  66 B
          fonts.css ⏤  235 B (+14 B)
          index.css ⏤  865 B (+6 B)
         popup.html ⏤  184 B
 window-provider.js ⏤  4.97 kB (+332 B)
 provider-bridge.js ⏤  5.98 kB (+105 B)


              ui.js ⏤  902 kB (+48.7 kB)
      background.js ⏤  596 kB (+3.13 kB)
   background-ui.js ⏤  66 B
          fonts.css ⏤  235 B (+14 B)
          index.css ⏤  865 B (+6 B)
         popup.html ⏤  184 B
 window-provider.js ⏤  4.97 kB (+332 B)
 provider-bridge.js ⏤  5.98 kB (+105 B)


              ui.js ⏤  902 kB (+48.7 kB)
      background.js ⏤  596 kB (+3.13 kB)
   background-ui.js ⏤  66 B
          fonts.css ⏤  235 B (+14 B)
          index.css ⏤  865 B (+6 B)
         popup.html ⏤  184 B
 window-provider.js ⏤  4.97 kB (+332 B)
 provider-bridge.js ⏤  5.98 kB (+105 B)


              ui.js ⏤  695 kB (-207 kB)
      background.js ⏤  443 kB (-154 kB)
   background-ui.js ⏤  0 B (-66 B)
 window-provider.js ⏤  3.42 kB (-1.55 kB)
 provider-bridge.js ⏤  3.63 kB (-2.34 kB)
          fonts.css ⏤  235 B
          index.css ⏤  865 B
         popup.html ⏤  184 B

✨  Done in 34.93s.
</pre>
</details>

**Pull request merged**